pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -549,6 +550,9 @@ func containerSucceeded(c *v1.Container, podStatus *kubecontainer.PodStatus) boo
 }
 
 func IsInPlacePodVerticalScalingAllowed(pod *v1.Pod) bool {
+	if runtime.GOOS == "windows" {
```

Comment:
This looks darn important!!!

Reply:
And ideally, this should be a runtime check. Isn't there something to mark a test as "does not work on Windows"?

Reply:
We may move this function `IsInPlacePodVerticalScalingAllowed` to  `pkg/kubelet/kuberuntime/kuberuntime_manager_linux.go` and (`pkg/kubelet/kuberuntime/kuberuntime_manager_windows.go` or  `kuberuntime_manager_others.go`).

Add `"[LinuxOnly]"` to the e2e test?

Reply:
Sounds worth trying, @pacoxu , @pohly @dims moving discussion [here](https://github.com/kubernetes/kubernetes/pull/128936#issuecomment-2505524573)